packages/bundle-source/test/sanity.js

```diff
@@ -105,7 +104,7 @@ test('getExport', async t => {
 
     const nestedEvaluate = src => {
       // console.log('========== evaluating', src, '\n=========');
-      return evaluate(src, { require, nestedEvaluate });
+      return evaluate(src, { nestedEvaluate });
```

Comment:
If we *did* want to pass `require` in, is there still a test that exercises that? I'm probably ok if we remove that feature entirely, but we should do it consciously, and if it stays in, we should test it.

Reply:
We need the `require` feature for deploy scripts.  It is tested during integration.

Reply:
I'll add a unit test for this, too.